src/angulartics-azure.js

```diff
@@ -0,0 +1,46 @@
+﻿/**
+ * @license Angulartics v0.17.2
+ * (c) 2013 Luis Farzati http://luisfarzati.github.io/angulartics
+ * Microsoft Azure Application Insights plugin contributed by https://github.com/anthonychu
+ * License: MIT
+ */
+(function (angular) {
+    'use strict';
+
+    /**
+     * @ngdoc overview
+     * @name angulartics.azure
+     * Enables analytics support for Microsoft Azure Application Insights (http://azure.microsoft.com/en-us/services/application-insights/)
+     */
+    angular.module('angulartics.azure', ['angulartics'])
+    .config(['$analyticsProvider', function ($analyticsProvider) {
+
+        $analyticsProvider.registerPageTrack(function (path) {
+            appInsights.trackPageView(path);
+        });
+
+        /**
+         * Numeric properties are sent as metric (measurement) properties.
+         * Everything else is sent as normal properties.
+         */

```

Comment:
@lunks Good point. I've thought about this and one idea was to provide a setting to specify a suffix to use to indicate a property is a metric. For example, if the suffix is `"metric"`...

```html
<button analytics-on analytics-category="Commands"  
    analytics-label="Button4 label" 
    analytics-numeric-value-metric="1">Button 4</button>
```

The provider will strip off the suffix before sending to Azure. And maybe by default the suffix is `null` and it'll automatically send numerics as metrics.

Do you have any other ideas to approach this? Thanks.